ENIGMAsystem/SHELL/Graphics_Systems/Direct3D11/Direct3D11Headers.h

```diff
@@ -17,8 +17,24 @@
 
 #ifndef ENIGMA_DIRECT3D11_HEADERS_H
 #define ENIGMA_DIRECT3D11_HEADERS_H
+
 #include <d3d11.h>
 #include <cmath>
 #include <string>
 using namespace std;
+
+namespace enigma {
+
+namespace dx11 {
+
+extern IDXGISwapChain* m_swapChain;
```

Comment:
I don't know why these are so widely needed, but okay.

Reply:
Well, the swap chain specifically didn't need to be exposed: 2ebc6d421c817663b1c4e04108e8963930423a02

Other than that, the device and device context are widely needed because they are literally how you make any call to the Direct3D11 API.